src/a.tex

```diff
@@ -517,8 +521,13 @@ \section{Atomic Memory Operations}
 elision~\cite{Rajwar:2001:SLE}.
 \end{commentary}
 
-The instructions in the ``A'' extension can also be used to provide
-sequentially consistent loads and stores.  A sequentially consistent load can
-be implemented as an LR with both {\em aq} and {\em rl} set. A sequentially
-consistent store can be implemented as an AMOSWAP that writes the old value to
-x0 and has both {\em aq} and {\em rl} set.
+The instructions in the ``A'' extension can be used to provide sequentially
+consistent loads and stores, but this constrains the implementation more
+than absolutely necessary.  A C++ sequentially consistent load can
```

Comment:
Maybe clarify that the constraints are performance constraints, and the implementation means hardware?

Reply:
Done.